doap_HUDI.rdf

```diff
@@ -46,6 +46,11 @@
         <created>2020-01-31</created>
         <revision>0.5.1</revision>
       </Version>
+      <Version>
```

Comment:
@yanghua let's update this step in the release guide?

Reply:
OK

Reply:
Have added a step in "Finalize the release" section:

```
Update DOAP file in the root of the project via sending a PR like this one.
```